src/components/expression/expression-panel.js

```diff
@@ -193,6 +200,7 @@ genotet.ExpressionPanel.prototype.initPanel = function() {
   this.container.find('#out').click(function() {
     if (this.data.zoomStack.length > 0) {
       var zoomStatus = this.data.zoomStack.pop();
+      genotet.logger.log('expression', 'expressionZoomOut', zoomStatus);
```

Comment:
expressionZoomOut seems to be redundant? will zoomOut be sufficient?

What is the result of logging a zoomStatus object?